### ISSUE TITLE
loss backward

### ISSUE THREAD

**Issue by mangdian** [ikostrikov/pytorch-a3c#28]
`(policy_loss + 0.5 * value_loss).backward()`
I am not familiar with pytorch. I am very confused about how the loss backward in two sub-net:actor and critic.because apparently that there are two output in the network(action and value),but(policy_loss + 0.5 * value_loss) seems to be loss for one sub-net.
Hope someone can help me.
Thank you so much.

**Comment by pfrendl** [comment]
`policy_loss` is the actor loss and `value_loss` is the critic loss. PyTorch Variables keep track of how they were calculated, and the backward call calculates the gradients for all model parameters with respect to the loss function, `(policy_loss + 0.5 * value_loss)` in this case.
